adagrams/game.py

```diff
@@ -1,11 +1,125 @@
+import random
+
 def draw_letters():
-    pass
+    letters = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    hand = []
+    for i in range(10):
+        choose_letter = list(letters.keys())
+        letter_choice = random.choice(choose_letter)
+        while letters[letter_choice] == 0:
+            letter_choice = random.choice(choose_letter)
+        hand.append(letter_choice)
+        letters[letter_choice] -= 1
+    return hand
+            
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    letters_used = [letter for letter in letter_bank]
```

Comment:
Another way to create a copy would be to use list slicing
`letters_used = letter_bank[:]`